docs/labels.md

```diff
@@ -1,9 +1,8 @@
 # Labels
 
 _Labels_ are key/value pairs that are attached to objects, such as pods.
```

Comment:
Please add after this sentence:

Labels are intended to be used to specify identifying attributes of objects that are meaningful and relevant to users, but which do not directly imply semantics to the core system.